lib/server.js

```diff
@@ -7,7 +7,7 @@ var _ = require('lodash');
 function initSync(mediator, mbaasApi, datasetId, syncOptions) {
   syncOptions = syncOptions || defaultConfig.syncOptions;
 
-  var dataListHandler = function(datasetId, queryParams, cb) {
+  var dataListHandler = function(datasetId, queryParams, metadata, cb) {
```

Comment:
@nialldonnellyfh I don't think we should maintain backwards compatibility.
I think we should do a major version bump to this module and let us use the major version bump of fh-mbaas-api.
From an external point of view (anyone using this module) it will be backwards compatible as long as they use the new version of fh-mbaas-api.

We could certainly hold off on merging this to master until `fh-mbaas-api@7.0.0` is published though.